src/main/scala/org/broadinstitute/dsde/workbench/sam/api/OldUserRoutes.scala

```diff
@@ -33,7 +32,7 @@ trait UserRoutes extends SamUserDirectives with SamRequestContextDirectives {
     })
   }
 
-  def userRoutes(samRequestContext: SamRequestContext): server.Route =
+  def oldUserRoutes(samRequestContext: SamRequestContext): server.Route =
```

Comment:
We should mark this as deprecated

Reply:
I think we should only mark as deprecated once all the functionality is implemented in the V2 routes